scopes/lanes/lanes/lanes.main.runtime.ts

```diff
@@ -16,6 +16,9 @@ import { CommunityAspect } from '@teambit/community';
 import type { CommunityMain } from '@teambit/community';
 import { Component } from '@teambit/component';
 import removeLanes from '@teambit/legacy/dist/consumer/lanes/remove-lanes';
+import { Lane } from '@teambit/legacy/dist/scope/models';
+import { Scope } from '@teambit/legacy/dist/scope';
```

Comment:
To avoid confusion, I'd suggest renaming it to `LegacyScope`. we do it in different places in Harmony code. 
```
import { Scope as LegacyScope } from '@teambit/legacy/dist/scope';
```